src/scripts/lambda/README.md

```diff
@@ -35,10 +35,20 @@ cd OLMo-core
 uv venv --python=3.12 /data/ai2/uv/OLMo-core-$USERNAME
 source "/data/ai2/uv/OLMo-core-${USERNAME}/bin/activate"
 uv pip install --torch-backend=cu129 numpy torch==2.8.0 torchvision torchaudio torchao==0.15.0
+uv pip install flash-attn --no-build-isolation
```

Comment:
This took like 5 hours for me lol, I probably should have ran it on a compute node.

Reply:
Oh really? I must have gotten the cached version after you installed it.